groundwork/patterns/gw_recipes_pattern.py

```diff
@@ -210,18 +216,23 @@ def __init__(self, name, path, plugin, description="", final_words=""):
         self.description = description
         self.final_words = final_words
 
-    def build(self, output_dir=None, **kwargs):
+    def build(self, no_input=False, extra_context=None, output_dir=None, **kwargs):
```

Comment:
Do not change the order of the attributes, othwerwise it is not backward compatible.

See https://github.com/useblocks/groundwork/pull/29/commits/848b9ddb1bc44e45a503fbd26928e3da97b3dcd4#diff-b53748b17deee9065571474b60adcad7R50 where you had to add output_dir so that test case is working again.

Reply:
Corrected.